### ISSUE TITLE
How realistic is the concern regarding an account ban ? Did anyone get banned for using it already ?

### ISSUE THREAD

**Issue by all-hail-the-omnissiah** [Sehyn/Wuwa_fpsunlocker#14]
How realistic is the concern regarding an account ban when editing the database with this tool ?

Im pretty happy with my account right now so im curious what everybody thinks about this and did anybody get banned already and suspects the editing to be the reason ?

**Comment by Sehyn** [comment]
Close to none it does not interact with the game executable only with a localfile on your pc

**Comment by spix18** [comment]
https://www.youtube.com/watch?v=tJL8CN_LpSg
they started banning people, carefull guys

**Comment by all-hail-the-omnissiah** [comment]
> https://www.youtube.com/watch?v=tJL8CN_LpSg they started banning people, carefull guys

thanks for mentioning this, i could only read the comments with translator so if you understand it, can you elaborate if the ban risk is resulting from the changes in the database or from use of this tool specifically ?

**Comment by spix18** [comment]
> > https://www.youtube.com/watch?v=tJL8CN_LpSg they started banning people, carefull guys
> 
> thanks for mentioning this, i could only read the comments with translator so if you understand it, can you elaborate if the ban risk is resulting from the changes in the database or from use of this tool specifically ?

the video and the upload has no real proof, he is just saying he got banned because he unlocked the fps from 60 to 120.
from our perspective this its a words and there is no proof of his claims, maybe he was rage cheating in tower ? noone will know for sure.
i just share the video to let you people know to think twice before risking their main account with good pulls for something that might bring later with patchs.

**Comment by maecry** [comment]
> > > https://www.youtube.com/watch?v=tJL8CN_LpSg they started banning people, carefull guys
> > 
> > 
> > thanks for mentioning this, i could only read the comments with translator so if you understand it, can you elaborate if the ban risk is resulting from the changes in the database or from use of this tool specifically ?
> 
> the video and the upload has no real proof, he is just saying he got banned because he unlocked the fps from 60 to 120. from our perspective this its a words and there is no proof of his claims, maybe he was rage cheating in tower ? noone will know for sure. i just share the video to let you people know to think twice before risking their main account with good pulls for something that might bring later with patchs.

tldr: false alarm, this user doesnt know what he talking about
